app/2.0/docs/about_20.md

```diff
@@ -541,3 +541,91 @@ If your project uses Polymer elements or behaviors, see [Polymer element availab
     `bower install`
 
 1.  See the [upgrade guide](upgrade) for information on getting your code working with 2.0.
+
+## Elements that support Polymer 2 {#elements-2}
+
+These elements have been updated to support Polymer 2.0, or require no updates. Click an element name to go to its `2.0-preview` branch on GitHub:
```

Comment:
Don't need to link to 2.0-preview branch, which will be obsolete. You can remove the `/tree/2.0-preview` from the URLs.